src/live_restore/live_restore_fs.c

```diff
@@ -603,8 +603,12 @@ __live_restore_fs_fill_holes_on_file_close(WT_FILE_HANDLE *fh, WT_SESSION *wt_se
 
     while ((hole = lr_fh->destination.hole_list_head) != NULL) {
```

Comment:
Maybe this is a suggestion for a future ticket but I think we should have some sort of progress messaging here. How many bytes/pages we've written and how many to go.

Reply:
Yeah there's future work here on statistics, we have planned stats around how many bytes are being written.